lib/name_drop/client.rb

```diff
@@ -37,9 +38,9 @@ def delete(endpoint)
 
     def request(method, endpoint, attributes = {})
       response = RestClient::Request.execute(create_request_hash(method, endpoint, attributes))
-      JSON.parse(response) unless response.empty?
+      JSON.parse(response).with_indifferent_access unless response.empty?
```

Comment:
Since `Base` is converting the attributes into `with_indifferent_access`, could we avoid doing it here too?